ui/src/components/TopBar/styles.module.css

```diff
@@ -22,3 +36,11 @@
   color: var(--primary-text);
   margin-left: 1vh;
 }
+@media only screen and (min-width: 600px) {
```

Comment:
700px

Reply:
@Aksh-Bansal-dev, can you define these breakpoints as global variables?

Reply:
> @Aksh-Bansal-dev, can you define these breakpoints as global variables?

I don't think this be done with vanilla CSS.
[reference](https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries).

Reply:
What's your opinion on using a CSS pre-processor? I think it will make the css dev easier.

Reply:
Yeah, that makes sense.

Reply:
For now, I set it to 700px. We will do it by scss variables after sass setup.

Reply:
@prajjwalkapoor, SASS setup is already done in #143

Reply:
@Samy-33 Yeah, I have replaced the values with sass variable.